opendevin/llm/llm.py

```diff
@@ -208,20 +222,12 @@ def wrapper(*args, **kwargs):
     def completion(self):
         """
         Decorator for the litellm completion function.
-        """
-        return self._completion
-
-    def do_completion(self, *args, **kwargs):
```

Comment:
Since this will be gone: we don't have any use-case for calling `completion` individually with args/kwargs?